app/controllers/RemoveFileController.scala

```diff
@@ -59,6 +61,7 @@ class RemoveFileController @Inject()(upscanService: UpscanService,
               }
             }.recover {
               case _ => {
+                PagerDutyHelper.log("removeFileUpscan", FILE_REMOVAL_FAILURE_UPSCAN)
```

Comment:
This PD doesn't really match the method name - you could put the class name and then the method name